coins/src/adapters/moneyMarkets/aave/aave.ts

```diff
@@ -56,23 +66,24 @@ async function getTokenPrices(chain: string, registry: string) {
   ]);
 
   let writes: write[] = [];
-  reserveData.map((r, i) =>
-    addToDBWritesList(
-      writes,
-      chain,
-      r.output.aTokenAddress.toLowerCase(),
-      undefined,
-      tokenInfo.decimals[i].output,
-      tokenInfo.symbols[i].output,
-      underlyingRedirects.filter((u) =>
-        u.PK.includes(r.input.params[0].toLowerCase())
-      )[0].redirect
-    )
-  );
+  reserveData.map((r, i) => {
+    try {
+      addToDBWritesList(
```

Comment:
why would this fail?

Reply:
because all the atokens redirect to the underlying, if the underlying isnt in the DB we cant direct it anywhere, so it'll error on finding 'redirect' on undefined

Reply:
its not an issue on aave but I guess it could be on forks